bin/compile

```diff
@@ -45,16 +44,16 @@ topic "Adding GPG keys"
 # This mimics the behavior of apt-key add to create a non-standard keyring and
 # import the datadog key into it.
 gpg --ignore-time-conflict --no-options --no-default-keyring --homedir $GPG_HOME_DIR --keyring $GPG_KEYRING_FILE --import $GPG_KEY_FILE | indent
-gpg --ignore-time-conflict --no-options --no-default-keyring --homedir $GPG_HOME_DIR --keyring $GPG_KEYRING_FILE --import $GPG_TRACE_KEY_FILE | indent
 
 topic "Updating apt caches"
 apt-get $APT_OPTIONS update | indent
 
 # Installing the Datadog Trace Agent will also install the Datadog Agent.
 topic "Fetching Datadog Agent and Trace Agent"
-apt-get $APT_OPTIONS -y --force-yes -d install --reinstall --no-install-recommends dd-trace-agent | indent
+apt-get $APT_OPTIONS -y --force-yes -d install --reinstall --no-install-recommends datadog-agent | indent
 
-topic "Extract deb packages"
+# Install the latest agent package.
+topic "Fetching deb packages"
 for DEB in $(ls -1 $APT_CACHE_DIR/archives/*.deb); do
```

Comment:
The is the one change from upstream. This is the official version https://github.com/DataDog/heroku-buildpack-datadog/blob/master/bin/compile#L57

And this is the issue I ran into before making the change: https://github.com/DataDog/heroku-buildpack-datadog/blob/master/bin/compile#L57